legal-api/requirements.txt

```diff
@@ -60,4 +60,4 @@ reportlab==3.6.12
 html-sanitizer==2.4.1
 lxml==5.2.2
 git+https://github.com/bcgov/business-schemas.git@2.18.39#egg=registry_schemas
-git+https://github.com/bcgov/lear.git#egg=sql-versioning&subdirectory=python/common/sql-versioning
+git+https://github.com/bcgov/lear.git#egg=sql-versioning&subdirectory=python/common/sql-versioning
```

Comment:
I don't think there is any change here. May want to remove this so that this doesn't kick off all the legal-api CI/CD stuff and confuse any existing/future issues with your PR

Reply:
Done, there wasn't supposed to be and I had reverted it but not the right way so I did now.